src/ngx_http_lua_shdict.c

```diff
@@ -1363,8 +1392,21 @@ ngx_http_lua_shdict_incr(lua_State *L)
             return 2;
         }
 
-        /* add value */
-        num = value + init;
+
+        if (flags == NGX_HTTP_LUA_SHDICT_INCR) {
+            /* add value */
+            num = value + init;
+        } else /* decr */ {
```

Comment:
Sorry, did you mean a newline above the `} else {`? I do see that previous commits. Please advise :)